docs/source/pillars/supporting.md

```diff
@@ -0,0 +1,217 @@
+
+```{list-table}
+:header-rows: 1
+:name: tab-procurement
+
+* - Statement
+  - Guidance
+  - Importance
+* -
+  -
+  -
+```
+
+## IT Service management
+
+_The implementation and management of quality IT services that meet the needs of the organisation._
+
+```{list-table}
+:header-rows: 1
+:name: tab-it-service-management
+
+* - Statement
+  - Guidance
+  - Importance
+* - You must have a team in place to support projects working with TREs
+  - This may be part of your organisation's IT support team, or separate. Responsibility should be clear and stakeholders should easily be able to access support appropriate to their needs
+  - Mandatory
+```
+
+## Relationship management
+
+_All activities aimed at ensuring a continuous level of engagement is maintained between the organisation and its customers, stakeholders & other interested parties._
+
+### Stakeholder relationships
+
+```{list-table}
+:header-rows: 1
+:name: tab-relationship-stakeholder
+
+* - Statement
+  - Guidance
+  - Importance
+* - You should have a clear process in place for stakeholders to feedback on your TRE infrastructure
+  - This may include a GitHub repository where people can open issues and discussions, communication streams like Slack or email, or forms stakeholders can fill in
+  - Recommended
+```
+
+### Patient and Public Engagement
+
+_What the organisation does to ensure it establishes and maintains mutually beneficial relationships between the organisation and the public in order to maintain trust._
+
+```{list-table}
+:header-rows: 1
+:name: tab-patient-public-engagement
+
+* - Statement
+  - Guidance
+  - Importance
+* -
+  -
+  -
+```
+
+## Legal services
+
+_The ability of the TRE organisation to access suitable and timely legal advice._
+
+<!-- Specific requirements? _e.g._ Article 32 of the GDPR requires organisations to regularly test and evaluate the effectiveness of the technical and organisational measures employed to protect personal data, and penetration testing is an effective way of assessing your technical defences. -->
+
+### Legal advisory
+
+_Ability of the organisation to provide suitable and timely legal advice._
+
+```{list-table}
+:header-rows: 1
+:name: tab-legal-services
+
+* - Statement
+  - Guidance
+  - Importance
+* -
+  -
+  -
+```
+
+### Data protection
+
+_Ability to ensure data is used fairly, lawfully and transparently. Used for specified, explicit purposes. used in a way that is adequate, relevant and limited to only what is necessary._
+
+```{list-table}
+:header-rows: 1
+:name: tab-legal-dp
+
+* - Statement
+  - Guidance
+  - Importance
+* -
+  -
+  -
+```
+
+### Contract management
+
+_What the organisation does to ensure that all contracts are effectively managed within required frameworks._
+
+```{list-table}
+:header-rows: 1
+:name: tab-legal-cm
+
+* - Statement
+  - Guidance
+  - Importance
+* -
+  -
+  -
```

Comment:
```suggestion
* - The organisation running the TRE should identify who will be responsible for managing contracts related to the TRE.
  - These contracts may include data sharing agreements, secondments of personnel or limitations on how results obtained with the data can be distributed.
  - Recommended
```

Reply:
Should this be helper text here: https://github.com/sa-tre/satre-specification/blob/main/docs/source/pillars/information_governance.md#policy-regulation-and-management

Reply:
Again, I *think* the difference is "do you have a policy for X?" vs. "do you have access to expert advice about X?"